### PR TITLE
Homework /day2

### PR DESCRIPTION
**Pull request by atheer10**
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
3/10
Was there anything that you struggled with?;
Generally had few syntax errors, sometimes certain functions didn't work (for ex .toLowerCase) 
Is there anything that you'd like some further information on?;
user's input 
Roughly how long did it take?
3 hours

### CODE REVIEW

**Comment by atheer10** [comment]
How difficult did you find this (out of 10)? 0 were no problems at all, 10 being impossible;
1/10
Was there anything that you struggled with?;
Again few syntax errors, also with the Math method I was a bit confused as to where to place it as in before or after but worked it out, also I couldn't really understand what's the use of return statement, however, I realized unless we actually do something with it it doesn't do anything .
  
Is there anything that you'd like some further information on?;
more on combined functions may be 

Roughly how long did it take?
1 hour

**Review by mfalthaw — APPROVED** [review]
Sweet work @atheer10 ! I loved how clean and organized your code was.

I agree, new topics like user input and functions in general can be challenging to wrap your head around it, but the more you use and experiment with them the better you’ll understand them. You can read more about user input and prompts [here](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt), and we'll definitely do more of this in class!

As for return statements, so basically they’re a way for us to get back values/results from the functions we called. It might not be very useful for this particular homework but they get more and more important as we build bigger projects. For example; say I’m building a website that has login and userAuthentication, and I have a function called `authenticateUser(username, password)` that returns `true` if _username_ AND _password_ are valid and `false` otherwise. So when a user wants to login, I wouldn’t be able to let them use the website until `authenticateUser(username, password)` returns `true` to that user. So as you can see return statements are **essential** when in comes Web Dev.

I've left some comments for you as well.

File: Atheer_Minwar/week_01/day_03/main.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+  //Percent of
+
+  function percentOf (numberOne ,numberTwo) {
+      var result = numberOne % numberTwo
+      console.log(numberOne + " is " +result+ "% of " +numberTwo);
+      return(result);
+  }
+percentOf(19,7);
```
[the remark this answers is not in the document]
Comment (atheer10) [reply Atheer_Minwar/week_01/day_03/main.js]: My bad ! 
Should have been the following : 

  function percentOf (numberOne ,numberTwo) {
      var result = ((numberOne / numberTwo) * 100); 
      console.log(numberOne + " is " + Math.round(result)+ "% of " +numberTwo);
      return(result);
  }
percentOf(2,4);

**Review by ghost — COMMENTED** [review]
Ather

**Pull request merged**